cms/static/js/views/overview.js

```diff
@@ -1,242 +0,0 @@
-define(["domReady", "jquery", "jquery.ui", "underscore", "gettext", "common/js/components/views/feedback_notification",
```

Comment:
There was something in this file that I would have had to change for the underscore upgrade (assuming the code ever executed... but I found that it did not). I think it is fine to have it part of this PR.